azext_iot/iothub/_help.py

```diff
@@ -316,6 +316,113 @@ def load_iothub_help():
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
 
+    helps[
+        "iot hub state"
+    ] = """
+        type: group
+        short-summary: Manage the state of an IoT Hub.
+        long-summary: For more information, see aka.ms/iothubstate
+    """
+
+    helps[
+        "iot hub state export"
+    ] = """
+        type: command
+        short-summary: Export the state of an IoT Hub to a file.
+        long-summary: |
+                       The exported state will include: arm template for hub, hub configurations (including ADM
+                       configurations and edge deployments), device information (including device identites,
+                       device twins, module identities and module twins).
+
+                       For more information, see aka.ms/iothubstate
+        examples:
+        - name: Export the supported state of the specified hub to the specified file.
+          text: >
+            az iot hub state export -n {iothub_name} -f {state_filename}
+        - name: Export the supported state of the specified hub to the specified file, overwriting the file contents.
+          text: >
+            az iot hub state export -n {iothub_name} -f {state_filename} -r
+        - name: Export only the devices and configurations of the specified hub to the specified file.
```

Comment:
Do you think the hint from the long summary that "devices" aspect includes module components is enough?

Reply:
modules are parts of devices - no? Isn't that implied to a degree? 

I can make this more specific:
Export only the devices (including associated modules) and configurations of the specified hub to the specified file.

but then the question becomes, do we also want to include wordings about twins, module twins, etc. I would say for now, the original help should be good enough

Reply:
Alright sure